src/FormlyMapper/FormlyField/BirthdayField.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Linio\DynamicFormBundle\FormlyMapper\FormlyField;
+
+use Linio\DynamicFormBundle\FormlyMapper\FormlyField;
+
+class BirthdayField extends FormlyField
+{
+    const ORDER_ASC = 'asc';
+    const ORDER_DESC = 'desc';
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getFieldType()
+    {
+        return 'birthday';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function buildFieldTypeConfiguration(): void
+    {
+        $this->formlyFieldConfiguration['templateOptions']['type'] = $this->getFieldType();
+        $options = $this->fieldConfiguration['options'];
+
+        $order = isset($options['order']) && (strtolower($options['order']) == (self::ORDER_ASC || self::ORDER_DESC))
+        ? $options['order']
+        : self::ORDER_DESC;
+
+        unset($this->formlyFieldConfiguration['templateOptions']['order']);
+
+        if (!isset($options['minAgeAllowed']) || !isset($options['maxAgeAllowed'])
```

Comment:
Shouldn't the checks be `<=` not `<` for 0? What happens if `maxAgeAllowed` is 0?

Reply:
I make the restriction that way to allow the current year, for example, if `minAgeAllowed = 0` and `maxAgeAllowed = 0` then only the current year will be allowed to enter ([0 => 2021]). Another example, if `minAgeAllowed = 1` and `maxAgeAllowed = 0` then the current year and the previous year will be allowed ([0 => 2020, 1 => 2021]).
Last example, if `minAgeAllowed = 1` and `maxAgeAllowed = 1` then the current year is no longer considered ([0 => 2020]). Do you think it's OK?

Reply:
I personally think that unset would be a better option than treating `0` as the current year. What does everyone else think?

To me this makes it more clear and doesn't hide logic behind a special value.